libs/npm/core/src/hooks/api/interfaces/IChartSectionSettingsModel.ts

```diff
@@ -9,10 +9,14 @@ export interface IChartSectionSettingsModel {
   groupBy: string;
   /** Override the chart title. */
   title: string;
-  /** Whether the chart is displayed horizontally. */
-  isHorizontal: boolean;
-  /** Whether the data value labels are visible. */
-  showDataValues: boolean;
+
+  /** CHART DISPLAY OPTIONS BELOW */
+
   /** Chart.JS configuration options */
   options: any;
+
+  showLegend?: boolean;
```

Comment:
I need to wire these up in the template generator eventually so that they can modify the output of a chart in each section.  Currently we don't support customizing these in a section.